go.mod

```diff
@@ -49,17 +47,15 @@ require (
 	go.opencensus.io v0.23.0
 	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
 	golang.org/x/sync v0.0.0-20201020160332-67f06af15bc9
-	golang.org/x/sys v0.0.0-20210225134936-a50acf3fe073 // indirect
 	google.golang.org/api v0.35.0 // indirect
 	google.golang.org/genproto v0.0.0-20210224155714-063164c882e6
 	google.golang.org/grpc v1.36.0
 	google.golang.org/protobuf v1.25.1-0.20201208041424-160c7477e0e8
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-	k8s.io/api v0.0.0-20191004102349-159aefb8556b // kubernetes-1.14.10
-	k8s.io/apimachinery v0.0.0-20191004074956-c5d2f014d689 // kubernetes-1.14.10
-	k8s.io/client-go v11.0.1-0.20191029005444-8e4128053008+incompatible // kubernetes-1.14.10
+	k8s.io/api v0.21.2 // kubernetes-1.14.10
+	k8s.io/apimachinery v0.21.2 // kubernetes-1.14.10
+	k8s.io/client-go v0.21.2 // kubernetes-1.14.10
```

Comment:
these comments are misleading

Reply:
I agree that e2e test should have their own mod files. I have been thinking of moving the e2e test out of the repo as they part of Open Match itself. We can provide it in another repo if necessary such as ecosystems. Was going to put the idea out there tomorrow during the contributors meeting